.github/CODEOWNERS

```diff
@@ -29,10 +29,10 @@
 /sdk/js-query/ @evan-gray @kev1n-peters @bruce-riley
 /sdk/js-wasm/ @evan-gray @kev1n-peters
 /sdk/js/ @evan-gray @kev1n-peters @panoel
-/sdk/rust/ @a5-pickle
+/sdk/rust/ @Daniel-Aaron-Bloom
```

Comment:
See error flagged by GitHub re lack of write access. Should we have a second codeowner for this path given Daniel is a new codeowner?